autopush/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.2.3'
+__version__ = '1.3.0dev'
```

Comment:
Why are we breaking the version format this way? I thought that we were simply going to increment. (I really don't want to get into the whole OAuth 1.0a v. 2.0 v.s 1.1.4 crap.)